Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -108,18 +109,26 @@ def get_incident_labels_map(labels):
     return labels_map
 
 
+def build_incident_fields_query(incident_data):
+    similar_keys_list = []
+    for key, value in incident_data.items():
+        str_value = str(value) if isinstance(value, int) else value
+        str_value = str_value.replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r")
+        str_value = str_value.encode('utf-8') if not isinstance(value, int) else str_value
+        similar_key = '{}:="{}"'.format(key, str_value) if isinstance(value, int) else '{}="{}"'.format(key, str_value)
```

Comment:
Let's split into two lines, with '{}:="{}"' and '{}="{}"' in a one-line if and a new variable

Reply:
done